generated_code/Point.py

```diff
@@ -62,14 +62,14 @@ def addKernels(generator, aderdg):
     mInvJInvPhisAtSources['k'] <= JInv * aderdg.db.M3inv['kl'] * basisFunctionsAtPoint['l'])
 
   #extract the moment tensors entries in SeisSol ordering (xx, yy, zz, xy, yz, xz)
-  assert(numberOfQuantities >= 6)
+  # assert(numberOfQuantities >= 6)
   momentToNRF_spp = np.zeros((numberOfQuantities, 3, 3))
   momentToNRF_spp[0, 0, 0] = 1
-  momentToNRF_spp[1, 1, 1] = 1
```

Comment:
Will need some generalization (at best, a move to the ADER-DG class).

For an ad-hoc implementation, how about checking the type of the equation? I.e. `if not isinstance(aderdg, AcousticMaterial)` or the likes?